pkgs/development/python-modules/kaleido/default.nix

```
@@ -97,7 +97,11 @@ buildPythonPackage rec {
     #ln -s ${lato}/share/fonts/lato/* $out/${python.sitePackages}/kaleido/executable/xdg/fonts/truetype/lato/
   '';
 
-  passthru.tests.kaleido = callPackage ./tests.nix { };
+  passthru.tests.kaleido = lib.optional stdenv.isLinux [
+    callPackage
+    ./tests.nix
+    { }
+  ];
```

This defines `passthru.tests.kaleido` as either a list with three entries (the function `callPackage`, `./tests.nix`, and `{ }`), or the empty list.

```suggestion
  passthru.tests = lib.optionalAttrs (!stdenv.isDarwin) {
    kaleido = callPackage ./tests.nix { };
  };
```

The ideal would be to only disable those tests that are actually broken, but maybe that‘s all of them.

---

Done. There's only one test, alas.

Thanks for the help.